dev-db/mysqltuner/mysqltuner-1.7.15.ebuild

```diff
@@ -0,0 +1,39 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+MY_PN=MySQLTuner-perl
+
+DESCRIPTION="MySQLTuner is a high-performance MySQL tuning script"
+HOMEPAGE="http://www.mysqltuner.com/"
```

Comment:
You could also add https://github.com/major/MySQLTuner-perl here.

Reply:
Done.